bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/common/BitcoindRpcClient.scala

```diff
@@ -58,6 +66,52 @@ class BitcoindRpcClient(val instance: BitcoindInstance)(
             new RuntimeException("Unexpected error when getting fee rate"))
       }
     }
+  // Chain Query Api
+
+  /** Gets the height of the given block */
+  override def getBlockHeight(
+      blockHash: DoubleSha256DigestBE): Future[Option[Int]] = {
+    getBlockHeader(blockHash).map(header => Some(header.height))
+  }
+
+  /** Gets number of confirmations for the given block hash */
+  override def getNumberOfConfirmations(
+      blockHash: DoubleSha256DigestBE): Future[Option[Int]] = {
+    getBlockHeader(blockHash).map(header => Some(header.confirmations))
+  }
+
+  /** Gets the number of compact filters in the database */
+  override def getFilterCount: Future[Int] = ???
```

Comment:
Why is this implemented here?

Reply:
So `BitcoindRpcClient` doesn't need to be `abstract`